starboard/loader_app/BUILD.gn

```diff
@@ -249,7 +255,7 @@ static_library("installation_manager") {
 }
 
 # TODO: b/309493306 - Stop building evergreen targets for all non-evergreen platforms.
-if (!is_host_win) {
+if (sb_is_evergreen_compatible && current_toolchain == starboard_toolchain) {
```

Comment:
maybe add !sb_is_modular here

Reply:
Added, good call. Even though it built somehow